data-imports/import-dob-permits.py

```diff
@@ -25,26 +25,26 @@
 
 dtype_dict = {
         'BOROUGH':                                'object',
-        'Bin #':                               'float64',
+        'Bin #':                               'object',
         'House #':                              'object',
         'Street Name':                            'object',
-        'Job #':                               'float64',
-        'Job Doc. #':                          'float64',
+        'Job #':                               'object',
+        'Job doc. #':                          'object',
         'Job Type':                               'object',
         'Self_Cert':                              'object',
-        'Block':                                 'float64',
+        'Block':                                 'object',
         'Lot':                                    'object',
         'Community Board':                        'object',
         'Zip Code':                               'object',
-        'Bldg Type':                             'float64',
+        'Bldg Type':                             'object',
```

Comment:
Should we even bother trying to import things as numbers instead of arbitrary strings? Should we do cleaning straight on the input csv file before importing into the db?